packages/gitmoji-changelog-cli/src/presets/node.js

```diff
@@ -1,17 +1,27 @@
 const readPkgUp = require('read-pkg-up')
+const pkgUp = require('pkg-up')
```

Comment:
What the difference between 'pkg-up' and 'read-pkg-up'. can we use only one ?

Reply:
`read-pkg-up` is built over `pkg-up`. I could use `read-pkg-up` to get path 😄